### ISSUE TITLE
Change highlight <mark> tag with a different one

### ISSUE THREAD

**Issue by rodica-andronache** [Codeinwp/otter-blocks#321]
as it's causing issues with WooCommerce as mentioned here https://github.com/Codeinwp/neve-pro-addon/issues/934

**Comment by HardeepAsrani** [comment]
@rodica-andronache But why? It's just a standard HTML tag after all. 😄

**Comment by selul** [comment]
Browsers it's adding a yellow bg by default to mark tags, we can replace with regular span tag or similar, in the same way, that core is doing.

**Comment by HardeepAsrani** [comment]
@selul That's exactly why we've used them. But my question is how it's causing an issue with Woo unless a user uses it themselves? Or am I missing something?

**Comment by selul** [comment]
The thing is that most of the themes style that mark tag with bg none due to that yellow bg by default. You can do some testing based on the top 20 themes from the repo. 

The woo thing is one reason why they style that with bg transparent.

**Comment by selul** [comment]
Another idea here is to add a distinct generic class to this mark, which could help on the theme side to target the reset.

**Comment by HardeepAsrani** [comment]
@selul There's nothing wrong with it as we anyway want styles to be decided by the theme. If the user wants they can override the theme defaults from Block settings and change the color. What do you think? We can change it to span tag if you want but I don't see anything wrong with themes styling mark as they want.

**Comment by selul** [comment]
Yep but the thing is because of the way that the majority of the theme are styling, the highlight mechanism is basically not working :) 
Let's add at least a distinctive class on the mark tag which can help target from the theme side.
